### PR TITLE
Remove unnecessary parameter in buildStatefulInliningInfo

### PR DESCRIPTION
**Pull request by zasdfgbnm**

### CODE REVIEW

**Comment by zasdfgbnm** [comment]
!build

File: csrc/id_model/id_model.cpp [hunk 1]
```diff
@@ -554,8 +553,8 @@ StatefulInliningInfo buildStatefulInliningInfo(
         auto consumer_tv_i = consumer_tvs.vector().at(i);
         auto all_consumer_i_ids = ir_utils::allIDsOf(consumer_tv_i);
 
-        auto sibling_map =
-            exact_graph.buildMapBetween(all_consumer_ids, all_consumer_i_ids);
+        auto sibling_map = permissive_graph.buildMapBetween(
```
Comment (zasdfgbnm) [inline csrc/id_model/id_model.cpp:556]: I don't think exact graph and permissive graph makes any difference in siblings.

**Comment by zasdfgbnm** [comment]
!build

**Comment by zasdfgbnm** [comment]
So building the loop graph does not anything about exact graph. Exact graph is related only for loop promotion?

**Review by naoyam — APPROVED** [review]
Makes sense. Thanks for the cleanup!

**Pull request merged**